### PR TITLE
Accepting website PRs.

### PR DESCRIPTION
**Pull request by TikhonJelvis**
[Rendered proposal](https://github.com/haskell-org/committee/blob/aa8db0b576f013ec129c1e50dfd7a8076a7cec21/proposals/0003-pr-process.md)

### CODE REVIEW

**Comment by TikhonJelvis** [comment]
I created this proposal based on the policy Rebecca and I proposed over email before we had this official proposals process. I figure this is good for transparency.

I ended up making some slight changes from our original email, which we can discuss in this thread.

File: proposals/0003-pr-process.md
Comment (rebeccaskinner) [inline proposals/0003-pr-process.md:18]: I might rephrase this slightly to say that all PRs must be open for no less than a full day after the last change before being merged regardless of when they are approved, in order to allow community feedback. As written it could be taken 
 to mean that we would need to wait an additional 24 hours after being approved, but I don’t think we need to couple the feedback period to the approval.

File: proposals/0003-pr-process.md
Comment (rebeccaskinner) [inline proposals/0003-pr-process.md:15]: Should we clarify major or controversial here? Ultimately it will be a matter of personal judgement but I think it might help to set expectations with some examples perhaps. 

Removing a link to a tool on the downloads page would be a good example to me of a controversial but not major change. 

Deprecating the wiki would be an example of a major but mostly in non-controversial change, but we could probably think of smaller examples as well.

File: proposals/0003-pr-process.md
Comment (rebeccaskinner) [inline proposals/0003-pr-process.md:13]: Should we waive the review requirement for maintainers in the case of an immediate breakage? Maybe we can add a line like:

For urgent issues impacting the security or availability of the site, maintainers may push a minimal fix without a review. Members of the committee should review any changes that were pushed without review as soon as possible to ensure a revert or follow up isn’t needed.

File: proposals/0003-pr-process.md
Comment (jaspervdj) [reply proposals/0003-pr-process.md:15]: Adding some examples would be good but ultimately it will be up to the committee members what's major or controversial on a case by case basis, I don't think we can come up with a perfect definition here.

File: proposals/0003-pr-process.md
Comment (jaspervdj) [inline proposals/0003-pr-process.md:4]: Needs a bump to `/3`

**Review by jaspervdj — COMMENTED** [review]
Thanks a lot!  Looks good to me -- @rebeccaskinner's comments are spot on, I don't have much else to add.

File: proposals/0003-pr-process.md
Comment (TikhonJelvis) [reply proposals/0003-pr-process.md:13]: For this one, let's keep the requirement as-is, and maybe also provide some way to ping the whole committee on Slack or something if we need a fast response.

If this requirement starts getting in the way, we can revisit it down the line.

File: proposals/0003-pr-process.md
Comment (TikhonJelvis) [reply proposals/0003-pr-process.md:15]: Makes sense. I added a couple of examples but, ultimately, all of these rules come down to judgement calls one way or the other.

File: proposals/0003-pr-process.md
Comment (TikhonJelvis) [reply proposals/0003-pr-process.md:18]: Good call.

**Comment by TikhonJelvis** [comment]
Thanks for the comments! I updated the proposal. Now that the holiday season is (mostly) over, do we want to take a vote on this?

**Comment by TikhonJelvis** [comment]
Given the timing, let's put the official vote off until we're done choosing new committee members. Could be a good way to get everyone into doing something concrete (even if small) right away.

**Comment by ryantrinkle** [comment]
This looks good to me.

File: proposals/0003-pr-process.md
Comment (tomjaguarpaw) [inline proposals/0003-pr-process.md:14]: I'm sceptical of the value of requiring non-author members to approve. Firstly if two committee members co-authored an uncontroversial PR then we already have reason to believe it will be a good PR. Secondly this rule is very easy to subvert by a committee member asking a conspirator non-committee member to submit the PR on their behalf. We don't want to cause so much frustration that subverting the rule becomes a tempting course of action!

Comment (tomjaguarpaw) [reply proposals/0003-pr-process.md:14]: And to add a bit of colour, suppose two committee members collaborate on producing a nice update to the website. There are only five other committee members. The policy as stated requires two of these five busy people to find time to review the PR. Reviews are *already* hard enough to come by. This particular policy would make the situation worse.

Comment (TikhonJelvis) [reply proposals/0003-pr-process.md:14]: Yeah. I've worked on a few PRs since I first wrote up this policy, and it definitely feels like it's both unnecessary *and* adds enough friction to slow PRs down substantially.

Comment (tomjaguarpaw) [reply proposals/0003-pr-process.md:14]: I suggest changing this to

```
2. Uncontroversial changes that affect the content of the site need two committee members to approve. If the PR was authored by one or more committee members those committee members are still allowed to be approvers.
```

**Review by tomjaguarpaw — APPROVED** [review]
Excellent. Love the new wording. Thank you (especially for getting it updated so quickly)!

**Comment by tomjaguarpaw** [comment]
(For those community members who are following the progress of this proposal, @TikhonJelvis's recent update will now be voted on by the committee using the Github PR review features.)

**Review by ryantrinkle — APPROVED** [review]

**Review by rebeccaskinner — APPROVED** [review]
I love the new content here and I think this is a great direction for us to move so that we can be more timely in making, and accepting, changes.

**Review by jaspervdj — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
We have 5/7 committee members approving this. Should we merge or should we ask Alexandre and Ida to look? (I don't know their Github handles otherwise I'd mention them in this comment).

**Comment by romefeller** [comment]
I also approve.

**Review by romefeller — APPROVED** [review]
I think we can merge.

**Comment by tomjaguarpaw** [comment]
@TikhonJelvis Would you like to do the honours of merging?

**Pull request merged**
